features/minion/repository/pom.xml

```diff
@@ -218,12 +218,15 @@
 
     <repositories>
         <repository>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-            <releases>
-                <enabled>true</enabled>
-            </releases>
+            <snapshots><enabled>true</enabled></snapshots>
```

Comment:
We can probably get rid of these changes now that jicmp has been released at version 2.0.0.